packages/office-ui-fabric-react/src/components/Dialog/Dialog.styles.ts

```diff
@@ -2,27 +2,44 @@ import { IDialogStyleProps, IDialogStyles } from './Dialog.types';
 import {
   IStyle,
   ITheme,
+  FontWeights,
+  ScreenWidthMinMedium,
 } from '../../Styling';
 
 export const getStyles = (
   props: IDialogStyleProps
 ): IDialogStyles => {
   const {
     className,
+    containerClassName,
+    contentClassName,
     theme,
+    hidden,
   } = props;
 
-  const { palette, semanticColors } = theme;
+  const dialogDefaultMinWidth = '288px';
```

Comment:
I think it might be better to make these a style prop you can pass the default in from base

Reply:
Agreed.